### PR TITLE
1466

### PR DESCRIPTION
**Pull request by JaumeFigueras**
Added today's rafagas. Docker composer is not working for me, so i couldn't test.

### CODE REVIEW

File: _posts/2020/2020-12-09-1466.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: rafaga
+date: 2020-12-09
+rid: 1466
+rafagas:
+
+- keyw: travelfromhome, books
+  desc: 'A literary travel agent recommends book sets matching your destination. It shows locations and traveling routes from the selected book set.'
```
Comment (jsanz) [inline _posts/2020/2020-12-09-1466.md]: This is a general comment.

Only one keyword can be used per link, so you need to remove one from each. You can still use the hashtag notation in the description for the removed, so for example in this case:

```suggestion
- keyw: books
  desc: 'A literary travel agent recommends book sets matching your destination. It shows locations and traveling routes from the selected book set #travelfromhome'
```

**Review by jsanz — CHANGES_REQUESTED** [review]
@JaumeFigueras you can test the output at your own version of the website published at

https://jaumefigueras.github.io/rafagas/

Apart from that, please just choose one keyword per link, thanks!!

File: _posts/2020/2020-12-09-1466.md (hunk 1 above)
Comment (JaumeFigueras) [reply _posts/2020/2020-12-09-1466.md]: Changes done!!

**Review by jsanz — APPROVED** [review]
looks good to me, thanks!! :shipit:

**Pull request merged**